peachjam/urls.py

```diff
@@ -13,11 +13,34 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from django.conf import settings
+from django.conf.urls.static import static
 from django.contrib import admin
 from django.contrib.auth import views
-from django.urls import include, path
+from django.urls import include, path, re_path
+from django.views.decorators.cache import cache_page
 
-urlpatterns = [
+from africanlii.feeds import (
```

Comment:
Peachjam shouldn't import anything from projects that depend on it; instead, pull this feed functionality into peachjam.